atlascope/core/models/tour.py

```diff
@@ -0,0 +1,33 @@
+# from typing import List
+from uuid import uuid4
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.gis.db import models as gisModels
+
+from django.db import models
+from rest_framework import serializers
+
+
+class Tour(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    investigation = models.ForeignKey(
+        'Investigation',
+        on_delete=models.CASCADE,
+        related_name='tour',
+    )
+    way_points = ArrayField(
+        ArrayField(
+            gisModels.PointField(null=True),
+            null=True
+        ),
+        null=True
+    )
+
+
+class TourSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tour
+        fields = '__all__'
+
+
+class TourCreateSerializer(serializers.Serializer):
+    way_points = serializers.ListField(child=serializers.ListField(child=serializers.IntegerField()))
```

Comment:
This specifically needs to be reworked because the IntegerField cannot be used here.

Reply:
Shouldn't it be a PointField serializer, instead of IntegerField?

Reply:
resolved in 2d813ecddf8363d972acfdf8626c85516203de80